inputFiles/thermoPoromechanics/ThermoPoroDruckerPrager_consolidation_smoke.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Included>
+    <File name="./ThermoPoroPlastic_consolidation_base.xml" />
+  </Included>
+
+  <Constitutive>
+    <PorousDruckerPrager
+      name="rock"
+      solidModelName="rockSolid"
+      porosityModelName="rockPorosity"
+      permeabilityModelName="rockPerm"
+      solidInternalEnergyModelName="rockInternalEnergy"
+    />
+
+    <DruckerPrager 
+      name="rockSolid"
+      defaultDensity="2700"
+      defaultBulkModulus="0.5e9"
+      defaultShearModulus="0.3e9"
+      defaultCohesion="0.1e6"
+      defaultFrictionAngle="15.27"
+      defaultDilationAngle="15.0"
+      defaultHardeningRate="10.0e6"
+      defaultDrainedLinearTEC="3e-7"/> 
+
+    <!-- SPHINX_POROSITY -->    
```

Comment:
do you use those `SPHINX` tags somewhere?

Reply:
No, this was copied from existing tutorial example

Reply:
i would remove them maybe

Reply:
Yes, just removed them